client/test/e2e/subsidies/subsidies.spec.js

```diff
@@ -0,0 +1,97 @@
+/* jshint expr:true */
+/* global element, by, beforeEach, inject, browser */
+
+var chai = require('chai');
+var expect = chai.expect;
+
+var FU = require('../shared/FormUtils');
+var helpers = require('../shared/helpers');
+var components = require('../shared/components');
+
+helpers.configure(chai);
+
+describe('Employees Module', function () {
```

Comment:
Please remember to change these labels.  This is not the "Employees Module".  It is the "Subsidy Module".

If a bug occurs, these labels will tell us where the problem is.  It is **extremely important** to make them indicate where the bug occurred.  I've already spent a long time debugging another API that was mis-labeled.